### PR TITLE
GCP Instructions on how to install and upgrade an Akka Cloud Platform using the command line

### PR DESCRIPTION
**Pull request by andreaTP**
References https://github.com/lightbend/akka-operator-helm/pull/25
and ultimately: https://github.com/lightbend/akka-platform-meta/issues/197

### CODE REVIEW

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc [hunk 1]
```diff
@@ -47,6 +49,95 @@ gcloud container clusters get-credentials acp-cluster-1 \
 ----
 Paste the command you copied from the Google console into a terminal to connect to the Kubernetes cluster. You are now ready to start using the Akka Cloud Platform. 
 
+== Deploy from command line
+
+To completely configure and use the Akka Cloud Platform you a few additional steps after switching to the Tab "Deploy via command line".
```
Comment (ignasi35) [inline docs-source/docs/modules/deployment/pages/gcp-install.adoc:54]:
```suggestion
To configure and use the Akka Cloud Platform you ??? a few additional steps after switching to the Tab "Deploy via command line".
```

I'd remove the `completely` (but that's a matter of taste. Then, the rest of the sentence looks incomplete and a bit too convoluted. Maybe something like:

```
To finish configuring the Akka Cloud Platform you must fulfill a few 
additional steps:

. On your browser, switch to the Tab "Deploy via command line".
. Generate a license key and save it to the disk (e.g. `license.yaml`). <--- should we indicate how to do this or link to the part of the docs where that is detailed?
etc...
```

I'm not sure my suggestion makes sense though, either.

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc [hunk 2]
```diff
@@ -47,6 +49,95 @@ gcloud container clusters get-credentials acp-cluster-1 \
...
+----
+kubectl apply -f license.yaml --namespace lightbend
+----
+
+. The output will be like (Take note of the secret name for future reference, in this case: `akka-cloud-platform-1-license`):
++
+[source,shell script]
+----
+secret/akka-cloud-platform-1-license created
+----
```
Comment (ignasi35) [inline docs-source/docs/modules/deployment/pages/gcp-install.adoc:77]:
```suggestion
. The output will be similar to the following snippet. Take note of the secret name for future reference, in the case below the name is: `akka-cloud-platform-1-license`.
+
[source,shell script]
----
secret/akka-cloud-platform-1-license created
----
```

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc [hunk 3]
```diff
@@ -47,6 +49,95 @@ gcloud container clusters get-credentials acp-cluster-1 \
...
+. Install the latest version with (using for `reportingSecret` the previously created secret name):
++
+[source,shell script]
+----
+helm install akka-operator akka-operator-helm/akka-operator \
+  --namespace lightbend \
+  --set provider.name=gcp \
+  --set reportingSecret=akka-cloud-platform-1-license
+----
+
```
Comment (ignasi35) [inline docs-source/docs/modules/deployment/pages/gcp-install.adoc:110]:
```suggestion
. Install the latest version with the command:
+
[source,shell script]
----
helm install akka-operator akka-operator-helm/akka-operator \
  --namespace lightbend \
  --set provider.name=gcp \
  --set reportingSecret=akka-cloud-platform-1-license  # <1>
----
<1> `akka-cloud-platform-1-license` is the previously created secret name (see steps above)
```

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc [hunk 4]
```diff
@@ -47,6 +49,95 @@ gcloud container clusters get-credentials acp-cluster-1 \
...
+== Upgrading a Akka Cloud Platform on GKE
```
Comment (ignasi35) [inline docs-source/docs/modules/deployment/pages/gcp-install.adoc:111]:
```suggestion
== Upgrading the Akka Cloud Platform on GKE
```

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc [hunk 5]
```diff
@@ -47,6 +49,95 @@ gcloud container clusters get-credentials acp-cluster-1 \
...
+. Install the latest version with (using for `reportingSecret` the previously created secret name):
++
+[source,shell script]
+----
+helm install akka-operator akka-operator-helm/akka-operator \
+  --namespace lightbend \
+  --set provider.name=gcp \
+  --set reportingSecret=akka-cloud-platform-1-license
+----
+
```
Comment (ignasi35) [inline docs-source/docs/modules/deployment/pages/gcp-install.adoc:140]: Same

**Review by ignasi35 — APPROVED** [review]
Left a few syntactic and copy comments. I am not familiar enough with the process to see if the described steps are correct.

**Comment by ignasi35** [comment]
I've approved early since I didn't see anything too big to be a blocker (though it be good to review some of the suggestions I made above)

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc (hunk 3 above)
Comment (andreaTP) [reply docs-source/docs/modules/deployment/pages/gcp-install.adoc:110]: chapeau asciidoc master

**Comment by andreaTP** [comment]
@ignasi35 thanks for the suggestions, ready for a new round :-)
this is my attempt to fix the first comment:
https://github.com/akka/akka-platform-guide/pull/700/commits/2b79cd1f81eadd3f05bda8fe304ef4195448bc7f

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc (hunk 5 above)
Comment (thothmoses) [reply docs-source/docs/modules/deployment/pages/gcp-install.adoc]: LGTM

**Review by thothmoses — COMMENTED** [review]
LGTM

**Pull request merged**